src/components/browse/components/ControlsAndResults.js

```diff
@@ -75,8 +75,10 @@ export class ControlsAndResults extends React.PureComponent {
             isOwnPage = true,         // <- False when rendered by EmbeddedSearchView, else is true when from a SearchView
             isContextLoading = false, // <- Only applicable for EmbeddedSearchView, passed in by VirtualHrefController only, else is false always since we initialize immediately over search-response context that already has first 25 results
 
-            // From EmbeddedSearchView/manual-entry, used if isOwnPage is true
-            maxHeight = SearchResultTable.defaultProps.maxHeight,
+            // From EmbeddedSearchView/manual-entry, used if isOwnPage is true,
+            maxHeight,
+            maxFacetsBodyHeight = SearchResultTable.defaultProps.maxHeight,
```

Comment:
@crfmc  It looks like `SearchResultTable.defaultProps.maxHeight `does not exist anymore. Is it ok that both `maxFacetsBodyHeight` and `maxResultsBodyHeight` are set as undefined?

Reply:
Thank you for pointing this out! Yes I was running into the same error on CGAP, I am adding maxHeight back to defaultProps.

Reply:
Should work properly now on CGAP and Fourfront